frontend/sozisel-app/src/components/SessionDetails/SessionCreation.tsx

```diff
@@ -0,0 +1,123 @@
+import "./SessionCreation.scss";
+
+import MuiAlert, { AlertProps } from "@material-ui/lab/Alert";
+import React, { ReactElement, useState } from "react";
+import {
+  useCreateSessionMutation,
+  useSessionTemplateQuery,
+} from "../../graphql";
+
+import CircularProgress from "@material-ui/core/CircularProgress";
+import MainNavbar from "../MainNavbar/MainNavbar";
+import SessionCustomization from "./SessionCustomization/SessionCustomization";
+import Snackbar from "@material-ui/core/Snackbar";
+import TemplateOverview from "./TemplateOverview/TemplateOverview";
+import { useLocation } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+function Alert(props: AlertProps) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export interface SessionCreationProps {
+  templateId: string;
+}
+
+export default function SessionCreation(): ReactElement {
```

Comment:
I don't get one thing. Why do we use names such `SessionCreation` but then we have `EditSession` and not `SessionEditing`? Those names are a little bit unfortunate. Ask @skusnierz for a better naming.